pages/dkp/kommander/2.2/dkp-upgrade/upgrade-konvoy/index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
@mwrightd2 we just merged a file for this with the exact same name. I don't know if when merging this one, the file would be overwritten or there might be conflicts? Just a heads-up. I would like to add 1 or 2 sentences, but don't know if to do that here...